tables/Pl-Pl-g1.utb

```diff
@@ -190,12 +191,10 @@ postpunc " 356
 
 prepunc ' 6-236
 postpunc ' 356-3
-postpunc '' 356
```

Comment:
Why is '' no longer needed?

Reply:
Just because I've no idea why it was there in the first place, there is nothing about it in the braille spec and this definition conflicts with back translations of quotation marks. The same goes for the entry removed below.

Reply:
OK

Reply:
@lukaszgo1 Are `''` and `&#x60;&#x60;` covered in your tests?

Reply:
No, they aren't. I've no idea what should be tested - I've removed these lines exactly because there is nothing special about them in the Polish braille.

Reply:
The fact that there is nothing special about them could be tested too :-)

The reason I ask is because someone obviously added this rule for a reason. Presumably the original author wanted to treat two single quotes as a double quote. I'm fine with changing it according to the real Polish rules, rather than trying to make up for wrong input. However I would feel better if this is covered by a test, so that the next person who comes along won't be able to change it back.